bld/namelist_files/namelist_definition.xml

```diff
@@ -6936,35 +6995,76 @@ Full pathname of dataset of O2 and 03 column densities above the model for look-
 Default: set by build-namelist.
 </entry>
 
-<entry id="aircraft_specifier" type="char*256(100)" category="cam_chem"
+<!-- aircraft h2o emissions -->
+
+<entry id="aircraft_h2o_datafile" type="char*256" input_pathname="abs" category="cam_chem"
        group="aircraft_emit_nl" valid_values="" >
-Filename of file that contains aircraft input file lists. The filenames in the files are relative
-to the directory specified by {{ hilight }}aircraft_datapath{{ closehilight }}.
-Default: set by build-namelist.
+Full pathname of the data file containing aircraft h2o emissions.
 </entry>
 
-<entry id="aircraft_datapath" type="char*256" input_pathname="abs" category="cam_chem"
+<entry id="aircraft_h2o_meshfile" type="char*256" input_pathname="abs" category="cam_chem"
        group="aircraft_emit_nl" valid_values="" >
-Full pathname of the directory that contains the files specified in
-{{ hilight }}aircraft_specifier{{ closehilight }}.
-Default: set by build-namelist.
+Full pathname of the ESMF mesh file corresponding to aircraft_h2o_datafile.
+</entry>
+
+<entry id="aircraft_h2o_year_first" type="integer" category="cam_chem"
+       group="aircraft_emit_nl" valid_values="" >
+First year of the aircraft_h2o_datafile to use.
 </entry>
 
-<entry id="aircraft_type" type="char*32" category="cam_chem"
-       group="aircraft_emit_nl" valid_values="CYCLICAL_LIST,SERIAL" >
-Type of time interpolation for data in aircraft aerosol files.
-Default: 'CYCLICAL_LIST'
+<entry id="aircraft_h2o_year_last" type="integer" category="cam_chem"
+       group="aircraft_emit_nl" valid_values="" >
+Last year of the aircraft_h2o_datafile to use.
 </entry>
 
-<entry id="aircraft_co2_file" type="char*256" input_pathname="abs" category="cam_chem"
-       group="camexp" valid_values="" >
-Full pathname of the ac_CO2 file specified in the filelist in
-{{ hilight }}aircraft_specifier{{ closehilight }}.  This is only to
-get this name into the cam.input_data_list for the CESM scripts.
-Default: set by build-namelist.
+<entry id="aircraft_h2o_year_align" type="integer" category="cam_chem"
+       group="aircraft_emit_nl" valid_values="" >
+Model year that aligns with aircraft_h2o_year_first.
+</entry>
+
```

Comment:
Please add `aircraft_h2o_tintalgo` back to the namelist:
```suggestion

<entry id="aircraft_h2o_tintalgo" type="integer" category="co2_cycle"
       group="co2_cycle_nl" valid_values="linear, lower, upper, nearest" >
Time interpolation algorithm to use for aircraft_h2o_datafile.
Default: linear
</entry>
```

Reply:
Done.